test/platform.test.ts

```diff
@@ -20,4 +22,223 @@ suite('Platform Tests', () => {
             done();
         });
     });
+
```

Comment:
no specific reason, I saw it somewhere else and I wanted to try it out. the test can be more readable this way

---
In reply to: [90500837](https://github.com/Microsoft/vscode-mssql/pull/436#discussion_r90500837) [](ancestors = 90500837)